tensorflow/python/keras/layers/core.py

```diff
@@ -655,12 +655,12 @@ class Lambda(Layer):
   [this guide](https://www.tensorflow.org/alpha/guide/keras/custom_layers_and_models) 
   for subclassing `tf.keras.layers.Layer`. 
   
-  The main reason to subclass `tf.keras.layers.Layer` instead using a 
-  `Lambda` layer is for saving and inspecting a Model. `Lambda` layers 
+  The main reason to subclass `tf.keras.layers.Layer` instead of using a 
```

Comment:
Fixed the grammar in the second paragraph.

The old paragraph reads:

> For more advanced use cases, subclassing `keras.layers.Layer` is preferred. One reason for this is that when saving a Model, `Lambda` layers are saved by serializing the Python bytecode, whereas subclassed Layers are saved via overriding their `get_config` method and are thus more portable. Models that rely on subclassed Layers are also often easier to visualize and reason about.

Reply:
Sorry I didn't notice this earlier. The last line is not grammatically correct and is confusing. Overall, I still feel that the old paragraph reads better than this one.

Reply:
You're right. I did put the last part back in.